src/components/AvatarWithIndicator.js

```diff
@@ -59,15 +59,20 @@ const AvatarWithIndicator = (props) => {
         isLarge ? styles.statusIndicatorLarge : styles.statusIndicator,
     ];
 
+    // If a policy was just deleted from Onyx, then Onyx will pass a null value to the props, and
+    // those should be cleaned out before doing any error checking
+    const cleanPolicies = _.filter(props.policies, policy => policy);
+    const cleanPolicyMembers = _.filter(props.policiesMemberList, member => member);
```

Comment:
Just wondering if using a compact which is a dedicated lodash function for this would not be better?? https://lodash.com/docs/4.17.15#compact

Or is there any of the falsey values which we need to account for?

Reply:
Tim answered here https://expensify.slack.com/archives/C07J32337/p1661786311194409?thread_ts=1661782875.058629&cid=C07J32337 no problem to use filter then